vllm/v1/engine/processor.py

```diff
@@ -48,16 +70,18 @@ def process_inputs(
         prompt: PromptType,
         params: Union[SamplingParams, PoolingParams],
         arrival_time: float,
+        max_logprobs: int,
```

Comment:
Just curious: when is this used?

Reply:
Somewhere in the code, we need to validate that the max_logprobs setting is valid (i.e. the setting which places an upper limit on the number of logprobs which a request can ask for.) I chose to put this check inside `process_inputs()` :

https://github.com/neuralmagic/vllm/blob/6ba743fe9a0b560991540b5c254edceac3605d36/vllm/v1/engine/processor.py#L83

which means that `process_inputs()` needs to have max_logprobs as an argument. The input processing method seemed like an appropriate place to check that an input is valid.